src/applications/simple-forms/40-0247/pages/veteranPersonalInfo.js

```diff
@@ -20,6 +22,9 @@ export default {
     veteranFullName: fullNameNoSuffixUI(),
     veteranDateOfBirth: dateOfBirthUI(),
     veteranDateOfDeath: dateOfDeathUI(),
+    // can't move this validation to Forms-Library web-component-patterns
+    // since one pattern can't detect presence of the other
+    'ui:validations': [dateOfDeathValidation],
```

Comment:
I didn't know you could place a ui:validation outside of the specific property like this. that's useful.  I see what you mean about it not being easy to fit into a pattern. one idea is to add an option to dateOfDeathUI to specify the dateOfBirth field. Maybe for now this is fine.  @ndsprinkle any thoughts on this?

Reply:
I'm surprised it works like that too, but that's actually pretty neat! I think this is fine for now, but I do agree it may make sense at some point to have a pattern that contains both of those fields with the shared validation or something like that.